src/CommunityToolkit.Maui/Alerts/Snackbar/Snackbar.android.cs

```diff
@@ -63,19 +63,19 @@ public async Task Show()
 		if (rootView is null)
 			throw new NotSupportedException("Unable to retrieve snackbar parent");
 
-		_nativeSnackbar = AndroidSnackbar.Make(rootView, Text, (int)Duration.TotalMilliseconds);
-		var snackbarView = _nativeSnackbar.View;
+		nativeSnackbar = AndroidSnackbar.Make(rootView, Text, (int)Duration.TotalMilliseconds);
+		var snackbarView = nativeSnackbar.View;
 
 		if (Anchor is not Page)
 		{
```

Comment:
it would be nice if we define the style for `if` blocks. somewhere we use braces (line 70), somewhere we skip them (line 64).

Reply:
Definitely! Great spot! Time for another vote I think